sdlib/d/gc/slab.d

```diff
@@ -23,6 +25,208 @@ struct SlabAllocGeometry {
 	}
 }
 
+struct SlabAllocInfo {
```

Comment:
Does this provides infos about the allocation, or does this actually manages the metatdata? It has setters, so i doubt it's just infos.